onnx/defs/data_type_utils.cc

```diff
@@ -112,7 +112,8 @@ const TypeProto& DataTypeUtils::ToTypeProto(const DataType& data_type) {
   std::lock_guard<std::mutex> lock(GetTypeStrLock());
   auto it = GetTypeStrToProtoMap().find(*data_type);
   if (GetTypeStrToProtoMap().end() == it) {
-    throw std::invalid_argument("Invalid data type.");
+    static const TypeProto empty_type_proto;
```

Comment:
Maybe add a comment to the header saying that if the input was invalid they will get an empty TypeProto back, and return_value.value_case() will return TypeProto::VALUE_NOT_SET.

Reply:
Thanks for the good suggestion! Addressed.